CHANGELOG.md

```diff
@@ -23,18 +23,16 @@ https://github.com/ansible/ansible-language-server/tree/main/docs/changelog-frag
 
 <!-- towncrier release notes start -->
 
-
 ## v0.5.1 (2022-03-01)
 
 ### Bugfixes
 
-* Ensure that Ansible calls do not return ANSI escapes, so we can parse them.
+- Ensure that Ansible calls do not return ANSI escapes, so we can parse them.
```

Comment:
@ssbarnea I thought you were going to fix the template before we start doing any releases, right?